app/jobs/registration_reminder_job.rb

```diff
@@ -2,10 +2,10 @@ class RegistrationReminderJob < ApplicationJob
   queue_as :default
 
   def perform(*args)
-    Event.where('start_time > ? and start_time < ?', Time.now, Time.now.tomorrow).each do |e|
-      EventMailer.reminder e
+    Event.where('start_time > ? and start_time < ?', Time.zone.now + 1.days, Time.zone.now + 2.days).each do |e|
+      EventMailer.delay.reminder(e)
       e.registrations.each do |r|
-        RegistrationMailer.reminder r
+        RegistrationMailer.delay.reminder(r)
```

Comment:
...and this mailer also sends an email to each attendee individually. I don't think we need this second one.

Reply:
Should this one also be `RegistrationMailer.reminder(r).deliver_later`?

Reply:
Yeah.